LICENSE

```diff
@@ -1,3 +1,13 @@
+# ''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''
+#  Copyright (c) 2024-2025 Mira Geoscience Ltd.                                '
+#                                                                              '
+#  This file is part of octree-creation-app package.                           '
+#                                                                              '
+#  octree-creation-app is distributed under the terms and conditions of the MIT License'
```

Comment:
could tweak copyright formatting in settings to have a box of 90 chars instead of 80

Reply:
done

Reply:
poetry config file for the copyright settings is supposed to be committed too

Reply:
this file:
![image](https://github.com/user-attachments/assets/54ef9ef5-8391-4a85-8870-a819c18733a9)